### ISSUE TITLE
solveFor() returns emtpy list in case the equation contains multiplication with one

### ISSUE THREAD

**Issue by hannaszepesi** [together-science/nerdamer-prime#54]
**How to reproduce?**

```
const solveConversion = (
  currentUnit: string,
  currentUnitValue: number,
  targetUnit: string,
  conversionString: string,
) => {
  const conversion = nerdamer(conversionString)
  const convertedExpression = conversion
    .evaluate({ [currentUnit]: currentUnitValue })
    .solveFor(targetUnit)
  const convertedValue = convertedExpression[0].text()
  return Number(convertedValue)
}
```

Call this function with the following parameters: 

`solveConversion('x', 10, 'y', 'x = 1 * y')`

Expected behaviour: The function returns 10.

If the same function is called with a different equation string e.g. 'x = 2 * y' it works.
Whenever we multiply by 1 convertedExpression variable will be an empty list.

**Comment by gunnarmein-ts** [comment]
.solveFor() does the job, but it doesn't always return an array (contrary to the docs). Check the result with
`if (!Array.isArray(convertedExpression)) {convertedExpression = [convertedExpression]}`
and your code should work.

**Comment by gunnarmein-ts** [comment]
@hannaszepesi did you try this with Nerdamer or Nerdamer-prime?

**Comment by hannaszepesi** [comment]
> @hannaszepesi did you try this with Nerdamer or Nerdamer-prime?

I have used Nerdamer, sorry if I should have created the issue it in that project. :)

**Comment by gunnarmein-ts** [comment]
No, this is the right place, Nerdamer is archived. Anyway, I hope you can use the work-around, I will fix this internally when I have a little time.

**Comment by gunnarmein-ts** [comment]
But you need to use Nerdamer-prime if you want me to look at anything, I won't be debugging the old Nerdamer anymore. Thanks!

**Comment by gunnarmein-ts** [comment]
Fixed in nerdamer-prime@1.1.18.
